en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
 
-YouTube’s structure as a video-sharing platform involves each video being accompanied by extensive user comments and discussions. While YouTube comments often take the form of short responses to videos and replies to other comments, their content and purpose can vary widely. Viewed as whole texts, they can frequently showcase broader ideological leanings, including the reaction to and effect of viewing a specific video.
+### YouTube and Scholarly Research
 
-Recent scholarship on YouTube's political dimensions has often explored the complicated problem of causation between viewing and changing perspectives, including qualitative sociological studies of YouTube users who have been purportedly radicalized through use of the platform. YouTube video comments represent a unique body of text, or a corpus, of discourse, describing how viewers receive and perceive politically charged messages, often from moving image media.  
+While YouTube has often been associated with entertainment, it is also a place where significant debates – explicit and implicit – play out between diverse demographics across the political spectrum. The platform therefore houses a wealth of culturally-relevant data that researchers and academics have begun to explore.
 
-Videos, including their titles, related metadata, and the video content itself, incite a reaction in viewers, where discussions and replies play out often for tens of thousands of comments. These comments often frame the future viewer's encounter with the video content, influencing subsequent viewers' thoughts, prompting some to share their interpretations in a reply or new comment even years after a video has been posted. 
+Built into YouTube’s structure as a video-sharing platform is a space for extensive user comments and discussions. While YouTube comments often take the form of short responses to the video and to other comments, their content and purpose can vary widely. Viewed as whole texts, they can frequently showcase broader ideological leanings, gleaned from the commenter's reaction to viewing a specific video.
 
-Comments appended to videos can influence future viewers’ encounters, and encounters between users can be days, months, and years apart. This means that the dialogue in comments may be an immediate back-and-forth between individuals, but also can involve extended hiatus and reactivation of discussion between a different group of participants. 
+Recent scholarship on YouTube's political dimensions has often explored the complicated problem of causation between viewing and changing perspectives: for example, qualitative sociological studies of YouTube users who have been purportedly radicalized through the platform. YouTube video comments represent a unique body of text – or corpus – of discourse, that describes how viewers receive and perceive politically charged messages, mostly from moving image media.  

```

Comment:
It would be nice to add an endnote here to reference one of these 'qualitative sociological studies'!